src/OpenTelemetry.Api/ActivityContextExtensions.cs

```diff
@@ -29,13 +31,27 @@ namespace OpenTelemetry
     public static class ActivityContextExtensions
     {
         /// <summary>
-        /// Returns a bool indicating if a ActivityContext is valid or not.
+        /// Returns a value indicating whether or not a <see cref="ActivityContext"/> is valid.
         /// </summary>
-        /// <param name="ctx">ActivityContext.</param>
-        /// <returns>whether the context is a valid one or not.</returns>
+        /// <param name="ctx"><see cref="ActivityContext"/>.</param>
+        /// <returns><see langword="true"/> if the context is a valid otherwise <see langword="false"/>.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [Obsolete("Use the IsValid method which accepts ActivityContext by reference for better performance.")]
         public static bool IsValid(this ActivityContext ctx)
         {
             return ctx != default;
         }
+
+        /// <summary>
+        /// Returns a value indicating whether or not a <see cref="ActivityContext"/> is valid.
+        /// </summary>
+        /// <param name="ctx"><see cref="ActivityContext"/>.</param>
+        /// <returns><see langword="true"/> if the context is a valid otherwise <see langword="false"/>.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsValid(this in ActivityContext ctx)
```

Comment:
This new one vs the old one:

|        Method |      Mean |     Error |    StdDev | Allocated |
|-------------- |----------:|----------:|----------:|----------:|
| IsValidLegacy | 2.4763 ns | 0.0107 ns | 0.0100 ns |         - |
|    IsValidRef | 0.1068 ns | 0.0049 ns | 0.0046 ns |         - |